dev/ts/shared/controls.ts

```diff
@@ -97,8 +97,8 @@ export function configureMousePointer(parameters: URLSearchParams): void {
 }
 
 export function configureControlsPanel(parameters: URLSearchParams): void {
-  const elControlsPanel = document.getElementById('controls-panel');
+  const elementControlsPanel = document.getElementById('controls-panel');
```

Comment:
**suggestion:** rename into `controlsPanelElement` for consistency with the type and the common usage.